### PR TITLE
[DASH-2403] Replace sockets view with sockets registry view

### PR DESCRIPTION
**Pull request by Flixow**

### CODE REVIEW

File: src/apps/Sockets/SocketsSummary.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          {`${SYNCANO_BASE_URL}/v1.1/instances/${item.instanceName}/endpoints/sockets/${call.name}/`}
+        </div>
+      </div>
+    ));
+  };
+
+  const renderCalls = (calls) => (
+    _.map(calls, (call, index) => (
+      <div
+        key={index}
```
Comment (patrykkopycinski) [inline src/apps/Sockets/SocketsSummary.js]: no `index` as a `key`

File: src/apps/Sockets/SocketsList.js [hunk 2]
```diff
@@ -1,88 +1,118 @@
...
+          styles={styles.detailsColumnHeader}
+        >
+          Details
+        </Column.ColumnHeader>
+      </ColumnList.Header>
+    );
+  },
+
+  render() {
+    const {
```
Comment (patrykkopycinski) [inline src/apps/Sockets/SocketsList.js]: it can be in one line

File: src/apps/Sockets/SocketsDialogStore.js [hunk 3]
```diff
@@ -1,13 +1,95 @@
...
+        this.sendSocketAnalytics(action, createdSocket);
+      },
+      error: () => this.trigger({ feedback: createdSocket.status_info, isLoading: false })
+    };
+
+    actions[createdSocket.status]();
+  },
+
+  onUpdateSocket() {
+    this.trigger({ isLoading: true });
```
Comment (patrykkopycinski) [inline src/apps/Sockets/SocketsDialogStore.js]: you can use mixin for that

File: src/apps/Sockets/SocketsDialogStore.js [hunk 4]
```diff
@@ -1,13 +1,95 @@
...
+    this.trigger({ isLoading: false });
+  },
+
+  onGetSocketCompleted(createdSocket, instanceName, action) {
+    const getSocket = _.debounce(Actions.getSocket, 500, { leading: true });
+    const actions = {
+      processing: () => getSocket(createdSocket.name, instanceName),
+      checking: () => getSocket(createdSocket.name, instanceName),
+      ok: () => {
+        this.trigger({ createdSocket, isLoading: false });
```
Comment (patrykkopycinski) [inline src/apps/Sockets/SocketsDialogStore.js]: trigger should be always at the end

File: src/apps/Sockets/SocketsDialogStore.js [hunk 5]
```diff
@@ -1,13 +1,95 @@
...
+  sendSocketAnalytics(type, payload) {
+    window.analytics.track('Used Dashboard Sockets API', {
+      type,
+      instance: payload.instanceName,
+      socketId: payload.name,
+      socket: ' socket '
+    });
+  },
+
+  getInitialState() {
```
Comment (patrykkopycinski) [inline src/apps/Sockets/SocketsDialogStore.js]: where is it used?

**Comment by Flixow** [comment]
image

**Comment by adamwardecki** [comment]
@gh3r @Flixow This task was about replacing the old socket list with new sockets. Search and filtering will be implemented later on. Also, we'll be removing an option to install sockets from the dash, so we can ignore the related bugs.

**Pull request merged**